cmd/discovery/main.go

```diff
@@ -28,3 +29,25 @@ func checkErr(err error) {
 		os.Exit(1)
 	}
 }
+
+// isEventDefined checks whether an event is defined in the bus.
+// It accepts strings or EventType, returns a boolean indicating that
+// the event was defined among the events emitted by the bus.
+func isEventDefined(i interface{}) bool {
```

Comment:
or move it to `github.com/kairos-io/kairos-sdk/kcrypt/bus` ? In case it's needed in other places.

Reply:
its been only used here forever. I think this was a more generic implementation t first but never went used anywhere other than here just for a single event so it doesnt even make sense to have it in a function IMHO.